### ISSUE TITLE
"Uncaught ReferenceError: require is not defined" in the build file 

### ISSUE THREAD

**Issue by nikhammerellis** [survivejs/webpack-book#254]
Noticed this after 5.2 in the book (it likely happened before and I wasn't paying attention), not seeing the Hello World text or proper styling in the browser, webpack seems to build correctly. 

In inspect, console tab is saying that "Uncaught ReferenceError: require is not defined" and where its not defined is strange, the app.js file that gets built from webpack. 

![screen shot 2017-04-08 at 12 49 07 pm](https://cloud.githubusercontent.com/assets/10385561/24831964/d26dec7e-1c59-11e7-9a8a-5599b82250f9.png)

Here is a link to my code up to 5.2 in a repo 
https://github.com/nikhammerellis/webpack-demo

Everything looks correct but I am new to webpack and this book has been my intro. 
I already deleted the app.js file and rebuilt it with npm run build and that didn't seem to work.

**Comment by bebraw** [comment]
@nikhammerellis How did you solve it? I haven't had a look yet. :)

**Comment by nikhammerellis** [comment]
@bebraw I ran through the beginning of the book (1-4) and repeated the steps in a new project and couldn't reproduce the error. I left out a particular better errors plugin this time. Based on the fact that it is working now, I have to assume I made one or more typos that somehow allowed webpack to compile incorrectly but successfully. That particular codebase is still jacked. I'm going with a freak coding accident for now, will let you know if the problem shows itself again.
